src/content/api/_code-snippets/src/invitations-create.yaml

```diff
@@ -0,0 +1,12 @@
+method: POST
```

Comment:
@MeganSteenkamp Why did you choose YAML?

Reply:
Did you consider JS modules? They'd be easier to manipulate and validate

```js
export default {
  path: '/api/invitations',
  headers: {
    X-Api-Key: '$api-key',
    Content-Type: 'application/json',
  },
  postData: {
    type: 'account-membership',
    resourceId: 'Hopo4g34sLVdjEMBs2p19F',
    resourceType: 'account',
    recipientAlias: 'user@org.com',
    params: {
      role: 'cashier',
    },
  }
};
```

Reply:
@tom-blake thanks I have updated this 🚀 
I had only thought about JSON and YAML and thought YAML is simpler but JS is even better.